golangci.yaml

```diff
@@ -25,8 +25,8 @@ linters:
     - unconvert
     - unused
     - varcheck
-    - whitespace
+    # - whitespace
```

Comment:
For some reason I had to do this to get the fixtures to lint properly...

Reply:
Might be related to how you are executing it. Using `make lint`?

Reply:
```shell
± zm |bp-homepage U:1 ✗| → make lint
> Installing golangci-lint...
cd tools; go install github.com/golangci/golangci-lint/cmd/golangci-lint
> Linting code...
inspect_builder_test.go:102: unnecessary trailing newline (whitespace)

					})
inspect_builder_test.go:150: unnecessary trailing newline (whitespace)

					})
make: *** [lint] Error 1
```

Reply:
You have some extra whitespace (empty lines). Try running `make format`.